examples/.eslintrc.cjs

```diff
@@ -0,0 +1,36 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+module.exports = {
```

Comment:
Oh, I see. The individual example configs extend both. No strong opinion here, but it seems like it would be a bit simpler / slightly less error prone to just have this extend the base config, and have the individual packages just extend this.

Reply:
I think the problem with this having a base is that the packages use different fluid configs. So if this were to have a base, then they would extend from this first and then they would override with the one they really want. But that would set these import rules to whatever is in the latest listed config.\
Unless we do programmatic generation, this appears to be the simplest config.
I am adding a comment to explain use.

Reply:
Good point. Programmatic config generation is something I'd like to look into in the future. We have a lot of variable axis for our packages that would be interesting to capture that way. But we should migrate to ESLint 9 first.